design/sketch.webidl

```diff
@@ -212,18 +212,17 @@ interface WebGPUPipelineLayout {
 };
 
 // BindGroup
-dictionary WebGPUBufferBinding {
+interface WebGPUBufferBinding {
```

Comment:
why is this an interface?

Reply:
I'm not sure what the difference is tbh. @litherum seemed to say WebGPUBinding should be an interface so I made this one an interface too.